composer.json

```diff
@@ -171,6 +171,7 @@
             "packages/Symfony/tests/Rector/FrameworkBundle/ContainerGetToConstructorInjectionRector/Source",
             "packages/NodeTypeResolver/tests/PerNodeTypeResolver/ParamTypeResolver/Source",
             "packages/NodeTypeResolver/tests/PerNodeTypeResolver/PropertyTypeResolver/Source",
+            "packages/Php74/tests/Rector/Property/TypedPropertyRector/Source",
```

Comment:
This can be removed, because it's already covered in `psr-4` here

Reply:
Oops, fixed.